tests/models/idefics/test_modeling_idefics.py

```diff
@@ -184,11 +229,30 @@ def get_config(self):
             type_vocab_size=self.type_vocab_size,
             is_decoder=False,
             initializer_range=self.initializer_range,
+            alpha_initializer=self.alpha_initializer,
             num_labels=self.num_labels,
             modality_type_vocab_size=self.modality_type_vocab_size,
             vision_config=self.vision_config,
         )
 
+    def model_last_hidden_states(
+        self,
+        model,
+        input_ids,
+        input_mask,
+        pixel_values,
+        image_attention_mask,
+        interpolate_pos_encoding,
+    ):
+        result = model(
+            input_ids,
+            attention_mask=input_mask,
+            pixel_values=pixel_values,
+            image_attention_mask=image_attention_mask,
+            interpolate_pos_encoding=interpolate_pos_encoding,
+        )
+        return result.last_hidden_state
```

Comment:
This helper function is a bit superfluous and makes the test harder to follow: it's better to have tests be verbose and explicit within the test itself, unlike normal code which we want to abstract. 

Just call
```
result = model(
            input_ids,
            attention_mask=input_mask,
            pixel_values=pixel_values,
            image_attention_mask=image_attention_mask,
            interpolate_pos_encoding=interpolate_pos_encoding,
        ).last_hidden_state
```
in the test